core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -594,6 +597,11 @@ object ConfigCommand extends Config {
     val brokerLogger = parser.accepts("broker-logger", "The broker's ID for its logger config.")
       .withRequiredArg
       .ofType(classOf[String])
+    // Tested via System Test kafkatest.tests.core.zookeeper_tls_test.ZookeeperTlsTest.test_zk_tls
```

Comment:
We don't tend to annotate code with pointers to tests. There are quite a few of these in the PR!